index.bs

```diff
@@ -320,9 +320,13 @@ the following [=policy-controlled features=]:
 
 Issue: Should we tie low-entropy-ness to allowlists, generally?
 
-Low entropy hint table {#low-entropy-table}
+Low-entropy hint table {#low-entropy-table}
 -------
-The <dfn export>low entropy hint table</dfn> below defines hints that are only exposing low amounts of entropy.
+The <dfn export lt="low-entropy hint table|low entropy hint table">low-entropy hint table</dfn>
+below defines hints that are only exposing low amounts of entropy.
```

Comment:
This is a little circular, is there any other possible reference?

Reply:
Any suggestions?

Reply:
Not seeing anything obvious https://github.com/search?p=1&q=org%3AWICG+entropy&type=Code

Maybe it's resolved by saying "defines hints *safe to send by default* due to their low amounts of entropy." That way there's some indication why it even matters.

Reply:
Good suggestion, thanks.